recipe/meta.yaml

```diff
@@ -1,51 +1,46 @@
-# Update with the following command
-#   conda skeleton pypi pyomo
-{% set version = "6.5.0" %}
+{% set version = "6.7.3" %}
+{% set name = "Pyomo" %}
 
 package:
-  name: pyomo
+  name: {{ name|lower }}
   version: {{ version }}
 
-source: # pypi md5
-  fn: Pyomo-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz
-  sha256: 5a23e775bba9fdbad22698fa1a841e662482edc979f2dea41cc6c54b1bb4b968
+source:
+  url: https://github.com/Pyomo/pyomo/archive/refs/tags/6.7.3.tar.gz
+  sha256: d31cdd1c63a80d39e51ff79b1e9dc10a1a0814fafe7f68777e682129e2179614
+
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --install-option=--with-distributable-extensions --no-deps -vv"
```

Comment:
Why did you remove `--install-option=--with-distributable-extensions`? This flag enables https://github.com/Pyomo/pyomo/blob/6.7.3/setup.py#L110-L131.

Reply:
I'm getting an error with it present: `no such option: --install-option`

Reply:
In these cases, we need to understand how that flag was used and how we can continue to use it.

In this specific case, you can pass it through an environment variable, see https://github.com/Pyomo/pyomo/blob/main/setup.py#L110-L113.

Reply:
Got it, thank you!

Reply:
`'--global-option="--with-cython --with-distributable-extensions"'` should work too, see https://github.com/Pyomo/pyomo/blob/6.7.3/.github/workflows/release_wheel_creation.yml#L56